source/Ship.cpp

```diff
@@ -988,6 +1008,22 @@ void Ship::Save(DataWriter &out) const
 			if(it.second)
 				out.Write("final explode", it.first->Name(), it.second);
 		});
+		if(hasHitEffect)
+		{
+		WriteSorted(hullHitEffects, effectSort, [&out](const EffectElement &it)
```

Comment:
How about some identitation here

Reply:
And the rest of the if clause obviously

Reply:
Yh, this needs indenting, the shield effect if clause as well needs it